switch_application.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
Can you add some more documentation here to explain the purpose and use case for this shell script please.

Reply:
updated documentation. 
I was hoping to not include the redis and rabbitmq into each other docker-compose files, instead we could just switch between tfrs and lcfs using this script while ensuring redis pod from LCFS container is active while tfrs application is running and rabbitmq from TFRS container is running while LCFS application is running.

Reply:
That's great thank you.